src/ServiceManager/Travely.ServiceManager.Abstraction/Interfaces/Repositories/IActivityTypeRepository.cs

```diff
@@ -1,8 +1,10 @@
-﻿using Travely.ServiceManager.Abstraction.Models.Db;
+﻿using System.Threading.Tasks;
+using Travely.ServiceManager.Abstraction.Models.Db;
 
 namespace Travely.ServiceManager.Abstraction.Interfaces.Repositories
 {
     public interface IActivityTypeRepository : IRepository<ActivityType>
     {
+        Task<ActivityType> GetActivityTypeByUniqueKeysAsync(long agencyId, string activityTypeName);
```

Comment:
The method have any argument called "key". What you mean by "ByUniqueKeys"?

Reply:
agree that the methods names' are inappropriate